projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/LawsuitActivity.java

```diff
@@ -347,5 +366,39 @@ public void onBackPressed() {
       viewPager.setCurrentItem(viewPager.getCurrentItem() - 1);
     }
   }
+
+  public void updateSharedPreferences() {
```

Comment:
I think it would be nicer to put this in an `AppSharedPreferences.java` class with static methods.

Reply:
I thought so too, wasn't sure if it's "enough" to create a dedicated class for it.
Changed it :+1: